ubi_manifest/worker/tasks/content_audit.py

```diff
@@ -120,12 +128,18 @@ def content_audit_task() -> None:
             # check that all content is up-to-date
             out_rpms_result = out_rpms.result()
             for in_rpm in _latest_input_rpms(in_rpms_fts):
-                if in_rpm.filename in in_mod_rpm_filenames:
+                if in_rpm.filename in modular_rpm_filenames:  # type: ignore [operator]
                     _LOG.debug(
                         "[%s] Skipping modular RPM %s", out_repo.id, in_rpm.filename
                     )
+                    name, version, _, _, _ = split_filename(in_rpm.filename)
```

Comment:
Is this wrongly copy-pasted?

Reply:
No. But I realize now that it's unnecessary, as RPM object has verison (used for stream) already.